### PR TITLE
Missed a constructor in AnalyticalGraphicsInc/cesium#6632.

### PR DESCRIPTION
**Pull request by emackey**
Addresses https://github.com/AnalyticalGraphicsInc/cesium/pull/6632#issuecomment-396773083

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@emackey, thanks for the pull request! Maintainers, we have a signed CLA from @emackey, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: Source/Scene/Instanced3DModel3DTileContent.js [hunk 1]
```diff
@@ -281,6 +283,7 @@ define([
             basePath : undefined,
             incrementallyLoadTextures : false,
             upAxis : content._tileset._gltfUpAxis,
+            forwardAxis : Axis.X,
```
Comment (lilleyse) [inline Source/Scene/Instanced3DModel3DTileContent.js]: This will need to get passed along through `ModelInstanceCollection` as well.

Comment (emackey) [reply Source/Scene/Instanced3DModel3DTileContent.js]: Is there a test case, in Sandcastle maybe?

Comment (lilleyse) [reply Source/Scene/Instanced3DModel3DTileContent.js]: The test tilesets all use boxes... but you can replace box.glb in [this tileset](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Specs/Data/Cesium3DTiles/Instanced/InstancedGltfExternal) with something else.

**Comment by lilleyse** [comment]
Looks good from my end -

**master**

before2

**1.46 and this PR:**

after2

File: Source/Scene/Instanced3DModel3DTileContent.js (hunk 1 above)
Comment (emackey) [reply Source/Scene/Instanced3DModel3DTileContent.js]: Thanks.  Was able to reproduce and confirm the fix here too.  Looks like travis failed, I'll look into that next.

**Comment by emackey** [comment]
No idea what's up with travis.

**Comment by mramato** [comment]
> No idea what's up with travis.

Travis is just flaky.  Looks like everything is passing now.

**Pull request merged**